src/Twilio/Rest/Client.php

```diff
@@ -166,12 +172,27 @@ public function __construct(string $username = null, string $password = null, st
             }
         }
 
+        if ($region) {
+            $this->region = $region;
+        } else {
+            if (\array_key_exists(self::ENV_REGION, $environment)) {
+                $this->region = $environment[self::ENV_REGION];
+            }
+        }
```

Comment:
Can this be refactored into a function so you can just do `$this->region = $this->getArg($region, self::ENV_REGION);`